src/controllers/user.ts

```diff
@@ -0,0 +1,48 @@
+import {User} from '../entity/User';
+import {getManager} from "typeorm";
+import {Request, Response} from "express";
+import {validate} from "class-validator";
+
+export const getUsers = async (req: Request, res: Response): Promise<void> => {
```

Comment:
Can't the return types be omitted here? Just for brevity.
```suggestion
export const getUsers = async (req: Request, res: Response) => {
```

Reply:
I think I can switch that linter setting off, by default it complains about it.

Reply:
Ah, makes sense

Reply:
This is part of the explicit-module-boundary-types rule, which is the one that requires explicit types for exported functions. If it ends up only really enforcing it on things that return Promise<void>, I'll probably disable that rule later.